src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AdminController.cs

```diff
@@ -122,8 +121,8 @@ public class AdminController : Controller
             {
                 Id = id,
                 DisplayName = application.DisplayName,
-                RedirectUri = application.RedirectUri,
-                LogoutRedirectUri = application.LogoutRedirectUri,
+                RedirectUri = application.RedirectUris.FirstOrDefault(),
```

Comment:
FYI I opened a follow-up ticket to track possible future improvements: https://github.com/OrchardCMS/OrchardCore/issues/1287.